flask_apispec/apidoc.py

```diff
@@ -93,7 +93,26 @@ def get_parameters(self, rule, view, docs, parent=None):
 
     def get_responses(self, view, parent=None):
         annotation = resolve_annotations(view, 'schemas', parent)
-        return merge_recursive(annotation.options)
+        options = []
+        for option in annotation.options:
+            exploded = {}
+            for status_code, meta in option.items():
+                if self.spec.openapi_version.major < 3:
+                    meta.pop('content_type', None)
+                    exploded[status_code] = meta
+                else:
+                    content_type = meta['content_type'] or 'application/json'
```

Comment:
@rusnyder I think it would be better if you use the get method to get the data so that it won't get errors when `content_type` does not exist in the `meta`. Something like this:

```
content_type = meta.get('content_type') or 'application/json'
```